src/js/actions/tools.js

```diff
@@ -755,7 +755,7 @@ define(function (require, exports) {
             }
         }
 
-        headlights.logEvent("tools", "mask-mode-" + currentLayer.kind);
```

Comment:
And if you take @shaoshing's suggestion to make the event be equal to the ID, make sure you to coerce that number into a string first.